airbnb/airbnb/Sources/Present/Detail/DetailViewController.swift

```diff
@@ -5,7 +5,191 @@
 //  Created by 김동준 on 2022/05/25.
 //
 
+import RxSwift
 import UIKit
 
-class DetailViewController: UIViewController {
+final class DetailViewController: UIViewController {
+    
+    private let imageSlider: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        return collectionView
+    }()
+    
+    private let backButton: UIButton = {
+        let button = UIButton()
+        button.layer.backgroundColor = UIColor.blue.cgColor
+        button.layer.cornerRadius = 25
+        return button
+    }()
+    
+    private let heartButton: UIButton = {
+        let button = UIButton()
+        button.layer.backgroundColor = UIColor.red.cgColor
+        button.layer.cornerRadius = 25
+        return button
+    }()
+    
+    private let shareButton: UIButton = {
+        let button = UIButton()
+        button.layer.backgroundColor = UIColor.yellow.cgColor
+        button.layer.cornerRadius = 25
+        return button
+    }()
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "asdfasdfdsa dafsdsafdsa fsaeklmek cfeiowjmrfeoiw crewo"
+        label.numberOfLines = 0
+        label.font = UIFont.systemFont(ofSize: 20)
+        return label
+    }()
+    
+    private let reviewLabel: UILabel = {
+        let label = UILabel()
+        label.text = "별 4.80(후기 127개)"
+        label.font = UIFont.systemFont(ofSize: 16)
+        return label
+    }()
+    
+    private let addressLabel: UILabel = {
+        let label = UILabel()
+        label.text = "서초구, 서울, 한국"
+        label.font = UIFont.systemFont(ofSize: 16)
+        return label
+    }()
+    
+    private let grayLine: UIView = {
+        let line = UIView()
+        line.layer.backgroundColor = UIColor.systemGray4.cgColor
+        return line
+    }()
+    
+    private let regidenceLabel: UILabel = {
+        let label = UILabel()
+        label.text = "레지던스 전체"
+        label.font = UIFont.systemFont(ofSize: 26)
+        return label
+    }()
+    
+    private let hostImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.layer.cornerRadius = 25
+        imageView.clipsToBounds = true
+        imageView.layer.backgroundColor = UIColor.black.cgColor
+        return imageView
+    }()
+    
+    private let viewModel: DetailViewModel
+    private let disposeBag = DisposeBag()
+    
+    init(viewModel: DetailViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+        bind()
+        attribute()
+        layout()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
조금 더 개선하자면, 어디서 호출하지 않는지 알려주면 좋지 않을까 생각합니다🙂

```suggestion
        fatalError("\(#function) init(coder:) has not been implemented")
```

Reply:
해당부분앞에 펑션키를 붙였습니다!